---a/net/minecraft/dispenser/ILocatableSource.java
+++b/net/minecraft/dispenser/ILocatableSource.java
@@ -1,5 +1,5 @@
 package net.minecraft.dispenser;
 
-public interface ILocatableSource extends ILocation
-{
-}
+public interface ILocatableSource extends ILocation {
+
+}
